packages/nextjs-kit/src/__mocks__/next/image.tsx

```diff
@@ -0,0 +1,4 @@
+export default function image(props: any) {
```

Comment:
Looks like there is a way to type this:

```suggestion
export default function image(
	props: React.DetailedHTMLProps<
		React.ImgHTMLAttributes<HTMLImageElement>,
		HTMLImageElement
	>,
) {
```